### DISCUSSION TITLE
Is libunwind supported on mac and windows?

### DISCUSSION THREAD

**Discussion by hexagonrecursion** [jeremy-rifkin/cpptrace#198] [Q&A]
Hello. I see a contradiction between your documentation and the documentation of libunwind. You claim to support libunwind as a backend on linux, on macos, on windows, and on mingw, but [libunwind only supports Linux, HP-UX, FreeBSD, QNX, and Solaris](https://github.com/libunwind/libunwind/blob/5ec31d584fe8716d32dbf353dc3f676fa1eca411/README#L23-L52). Could you please clarify? Is your documentation wrong? Is libunwind documentation wrong? Am I looking in the wrong plance?

**Comment by jeremy-rifkin** [comment]
Hi,
I'm not sure. I think I have tested on macos with libunwind before with https://formulae.brew.sh/formula/libunwind, but it looks like macos+libunwind isn't part of my current CI checks.
